### PR TITLE
MM-34288: Add a deployment sync command

### PR DESCRIPTION
**Pull request by agnivade**
Several times, it is necessary to run a terraform sync
to synchronize any changes made manually to the AWS cluster.

It is difficult to run it manually because one has to pass
all the parameters too. Hence, a separate command will be useful.

https://mattermost.atlassian.net/browse/MM-34288

### CODE REVIEW

File: deployment/terraform/utils.go [hunk 1]
```diff
@@ -173,3 +173,30 @@ func (t *Terraform) getStatePath() string {
 	}
 	return statePath
 }
+
+func (t *Terraform) getParams() []string {
```
Comment (isacikgoz) [inline deployment/terraform/utils.go:177]: Could we use this in `*Terraform.Destroy()`?

**Review by isacikgoz — COMMENTED** [review]
Awesome charity work 💯 Just a question below.

File: deployment/terraform/utils.go (hunk 1 above)
Comment (agnivade) [reply deployment/terraform/utils.go:177]: Oops, totally missed that one. I swear I checked the code :stuck_out_tongue:

**Comment by agnivade** [comment]
I would really love the spread operator to be used in between. I'm pretty sure there's a proposal for it somewhere :P

File: cmd/ltctl/main.go [hunk 2]
```diff
@@ -47,6 +47,16 @@ func RunInfoCmdF(cmd *cobra.Command, args []string) error {
...
 }
 
+func RunSyncCmdF(cmd *cobra.Command, args []string) error {
+	config, err := getConfig(cmd)
+	if err != nil {
+		return err
+	}
+
+	t := terraform.New("", config)
+	return t.Sync()
```
Comment (streamer45) [inline cmd/ltctl/main.go:57]:
```suggestion
	return terraform.New("", config).Sync()
```

**Review by streamer45 — APPROVED** [review]
This is a highly welcome addition :tada: Thanks :+1: 

Need to fix that `"fmt" imported but not used` in `destroy.go` but otherwise looks great.

**Review by isacikgoz — APPROVED** [review]
🙏

**Pull request merged**